config/env_sample.hjson

```diff
@@ -52,7 +52,7 @@
     # default password length indicator incase of creating user from command line or LTI auto login
     "RANDOM_PASSWORD_DEFAULT_LENGTH": 32,
     # Enable secure cookies, also set your trusted origin (example of instructure.com)
-    "CSRF_COOKIE_SECURE": false,
+    "CSRF_COOKIE_SECURE": true,
```

Comment:
@jonespm, any concerns here? I think I had to change this when following Getting Started to get it to work, so I figured we may as well make it the default.

Reply:
Are you sure this worked with localhost in Getting Started? 

According to Django setting this means that browsers have to have [SSL connections](https://docs.djangoproject.com/en/4.1/ref/settings/#csrf-cookie-secure) which we don't have locally, only if using ngrok/loophole. The slight problem with ngrok/loophole is they're slower in dev because of the huge size of the development mode javascript. So I'm guessing you don't want to use them unless you have to?

We could include some steps to generate a self signed cert for Gunicorn. We coould also say you have to use ngrok or loophole locally with or without LTI. We could also remove the LTI local development entirely like we have for our other apps, but that makes it hard to contribute. 

This is kind of where we need a few guides that will have different settings. 

1. Local Developers with local data with and without a proxy
2. Running in production on Openshift with Unizin Data
3. Running in production in some other cloud environment (like GCP) with Unizin Data

And probably don't have anything for non-Unizin Data. I'd probably change this back to false, but we should make sure it's mentioned why this would be changed?

Reply:
Okay, I thought I did without LTI, but I will check.

Reply:
Seems like I was mistaken here, it worked with `false`. I've changed it back and added a comment. https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1398/commits/0826f71869fb619b6d3c740682df4d0135839673